src/util/make-range-transform.test.js

```diff
@@ -33,17 +43,71 @@ describe('makeRangeTransform', () => {
             clampOutput: true,
         })
         expect(transformFunction2(103)).toBeCloseTo(4.41, 2)
+        const transformFunction3 = makeRangeTransform({
+            domain: [93, 117],
+            range: [0, 10],
+            clampOutput: true,
+        })
+        expect(transformFunction3(60)).toBe(0)
+        const transformFunction4 = makeRangeTransform({
+            domain: [97, 114],
+            range: [3, 7],
+            clampOutput: true,
+        })
+        expect(transformFunction4(150)).toBe(7)
+    })
+    
+    test('should return the inverse of the value for inverted range and domain with clampOutput true', () => {
+        const transformFunction1 = makeRangeTransform({
+            domain: [100, 0],
+            range: [0, 10],
+            clampOutput: true,
+        })
+        expect(transformFunction1(80)).toBe(2)
+        const transformFunction2 = makeRangeTransform({
+            domain: [0, 100],
+            range: [10, 0],
+            clampOutput: true,
+        })
+        expect(transformFunction2(80)).toBe(10)
+        const transformFunction3 = makeRangeTransform({
+            domain: [100, 0],
+            range: [10, 0],
+            clampOutput: true,
+        })
+        expect(transformFunction3(80)).toBe(10)
```

Comment:
Yeah, I was a bit surprised by that as well when I first wrote the tests with skewered values. I actually made the values a bit more apparent so that I it was obvious such a weird thing was taking place.